src/error.rs

```diff
@@ -42,6 +42,8 @@ pub enum CantDoReason {
     OutOfRangeSatsAmount,
     /// For users trying to do actions on dispute that are not theirs
     IsNotYourDispute,
+    /// For solvers when admin has taken over their dispute
+    DisputeTakenByAdmin,
```

Comment:
I think we should have a more generic error type in case the dispute is taken, `DisputeAlreadyTaken` works for a dispute taken by the admin or any solver